app/components/Table.tsx

```diff
@@ -28,6 +28,7 @@ export default function Table<T extends Record<string, unknown>>({
           {columns.map((col) => (
             <th
               key={String(col.path)}
+              className="text-nowrap"
```

Comment:
opinionated style change here in that i strongly dislike when column headers wrap. the downside of this is that table columns are wider, but they are easier to read

Reply:
fwiw between column wrap and sidescrolling, column wrap is the way less annoying one to me. but maybe we headers dont wrap and column contents do? idk but also respect your preferences, best of both worlds would be that everyone can adjust their tables just how they like them like in moped :stuck_out_tongue_closed_eyes:

Reply:
+1 configurable columns would be a really nice enhancement. i find these crash dates and locations impossible to read when they're wrapped 😵‍💫 

<img width="706" alt="Screenshot 2025-01-29 at 1 38 14 PM" src="https://github.com/user-attachments/assets/95948f48-f06a-4d5c-8c85-071227fe8580" />

Reply:
yeah thats so fair, they are much harder to read! im  down to keep it how you have it for now and explore configurable columns in the future/see if the vz team have any painpoints w the scrolling once the mvp is shipped

Reply:
Can you side scroll only using a mouse? I hope the VZ team uses their trackpads or know to scroll all the way down to see the horizontal scroll bar

Reply:
i believe there are two ways to side-scroll with a mouse:
- drag with the scroll wheel
- old shift and scroll the wheel

Reply:
shift!!! TIL